src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -955,6 +960,15 @@ void ClusterResourceScheduler::UpdateLastResourceUsage(
 }
 
 void ClusterResourceScheduler::FillResourceUsage(rpc::ResourcesData &resources_data) {
+  // Update local object store usage and report to other raylets.
```

Comment:
Is this change related?

Reply:
This is #20026, but it's reverted because somehow it made 1tb autoscaler shuffling timed out. But when it's combined with this PR it worked so I just redo it here since it's small.